akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md

```diff
@@ -1,6 +1,6 @@
 # ActorFlow.ask
 
-Use the `AskPattern` to send each element as an `ask` to the target actor, and expect a reply back that will be sent further downstream.
+Use the "Ask Pattern" to send each stream element as an `ask` to the target actor (of the new actors API), and expect a reply back that will be emitted downstream.
```

Comment:
```suggestion
Use the "Ask Pattern" to send each stream element as an `ask` to the target actor (of the Typed Actors API), and expect a reply back that will be emitted downstream.
```

Reply:
We try to use "new actors API" consistently.